.circleci/config.yml

```diff
@@ -1,94 +1,261 @@
-# Python CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-python/ for more details
-#
-version: 2
+# Note that the following stanza uses CircleCI 2.1 to make use of a Reusable Executor
+# This allows defining a docker image to reuse across jobs.
+# visit https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-executors to learn more.
+# NixOS stuff taken from
+# https://github.com/PrivateStorageio/PaymentServer/blob/master/.circleci/config.yml
 
-jobs:
+version: 2.1
+
+orbs:
+  check_untracked_changes: niteo/check-untracked-changes@1
+
+workflows:
+  version: 2
+
+  workflow:
+    jobs:
+      - Checkout
+      - Backend Deps:
+          requires:
+            - Checkout
```

Comment:
`Checkout` and `Backend Deps` can probably be merged into a single job, saves time on cache upload/download.

Reply:
Moreover, we don't need separate jobs here. The steps are fast enough to run them in a single job.

Reply:
Done.